runtime/queries/clojure/highlights.scm

```diff
@@ -0,0 +1,69 @@
+
+;; other calls
+(list_lit
+ .
+ (sym_lit) @function)
+
+;; interop-ish
+(list_lit
+ .
+ (sym_lit) @function.method
+ (.match? @function.method "^\\."))
+
+;; other symbols with dots
+((sym_lit) @variable
+ (.match? @variable "\\."))
+
+;; quote
+(quoting_lit) @constant.character.escape
+
+;; syntax quote
+(syn_quoting_lit) @constant.character.escape
```

Comment:
There are some extra highlights we could add like

```scm
["{" "}" "(" ")" "[" "]" "#"] @punctuation.bracket
["~" "~@" "#'" "@"] @operator
```

for the punctuation and some of the quoting stuff